packages/target-decisioning-engine/src/geoProvider.js

```diff
@@ -108,7 +109,7 @@ export function GeoProvider(config, artifact) {
     }
 
     // When ipAddress is the only geo value passed in to getOffers(), do IP-to-Geo lookup.
-    const geoLookupPath = getGeoLookupPath(config);
+    const geoLookupPath = `https://${CDN_BASE_PATH}/v${SUPPORTED_ARTIFACT_MAJOR_VERSION}/geo`;
```

Comment:
Let's move this to top of file and name it GEO_LOOKUP_URL since it's a static constant now.